### PR TITLE
dev-ml/ocaml-cstruct: new ebuild

### PR DESCRIPTION
**Pull request by tomboy-64**
- needed as a transitive dependency of Libre S3 1.1 (net-misc/libres3)
- explicit support for MirageOS is not enabled

@aballier

### CODE REVIEW

File: dev-ml/ocaml-cstruct/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <herd>ml</herd>
+    <herd>proxy-maintainers</herd>
+    <maintainer>
+        <email>tomboy64@sina.cn</email>
+    </maintainer>
+        <use>
```
Comment (mgorny) [inline dev-ml/ocaml-cstruct/metadata.xml]: Once again, you use different indent for <use> than for other elements.

**Comment by idella** [comment]
@gentoo/proxy-maint

File: dev-ml/ocaml-cstruct/metadata.xml (hunk 1 above)
Comment (tomboy-64) [inline dev-ml/ocaml-cstruct/metadata.xml]: hmm. i figured i fixed this.
gonna check that out. multiple branches are really annoying with this kind of mass-replace.

**Comment by aballier** [comment]
please move ocaml to RDEPEND and it's good to go

File: dev-ml/ocaml-cstruct/ocaml-cstruct-1.7.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+RDEPEND="
+	async? ( dev-ml/async:= )
+	camlp4? ( dev-ml/camlp4:= )
+	lwt? ( dev-ml/lwt:= )
+	dev-ml/ocplib-endian:=
+	dev-ml/sexplib:=
+	dev-ml/type-conv:=
+"
+DEPEND="
+	>=dev-lang/ocaml-4.01:=
```
Comment (mgorny) [inline dev-ml/ocaml-cstruct/ocaml-cstruct-1.7.0.ebuild]: `:=` does not make sense in `DEPEND` alone ;-).

**Comment by idella** [comment]
@gentoo/proxy-maint

**Comment by tomboy-64** [comment]
please don't merge yet; still checking dependency on use-flags.

**Comment by tomboy-64** [comment]
@aballier good to go.

**Comment by aballier** [comment]
merged, thanks!

**Pull request merged**